apisix/plugins/zipkin.lua

```diff
@@ -44,6 +55,11 @@ local _M = {
 
 
 function _M.check_schema(conf)
+
+    if not conf.server_addr or conf.server_addr == '' then
+        conf.server_addr = ngx.var.server_addr
```

Comment:
It seems wrong. I think we only can get the correct server address in `rewrite/access` phase.

Reply:
sure, how about move it into  create_tracer  function, like 
```
    if not conf.server_addr or conf.server_addr == '' then
        conf.server_addr = ngx.var.server_addr
    end
```

Reply:
I think it is wrong too. 

we need to update this line:

https://github.com/apache/incubator-apisix/blob/e36a21e187dd2016f49c93dca326cb4f6d17c928/apisix/plugins/zipkin.lua#L99

```lua
-- generate different tracers for different "server_addr" values

local tracer = core.lrucache.plugin_ctx(plugin_name .. "#" .. ctx.var["server_addr"], ctx,
                                            create_tracer, conf)
```

Reply:
ok